static/app/views/releases/detail/overview/releaseComparisonChart/index.tsx

```diff
@@ -1040,20 +1048,52 @@ const ChartTable = styled(PanelTable)`
   border-top-right-radius: 0;
 
   @media (max-width: ${p => p.theme.breakpoints[2]}) {
-    grid-template-columns: min-content 1fr 1fr 1fr;
+    grid-template-columns: repeat(4, minmax(min-content, 1fr));
   }
 `;
 
-const Cell = styled('div')<{align: 'left' | 'right'}>`
-  text-align: ${p => p.align};
+const Cell = styled('div')<{role?: ComparisonRow['role']}>`
+  text-align: right;
   ${overflowEllipsis}
+  ${p =>
+    p.role &&
+    ['parent', 'children'].includes(p.role) &&
+    css`
+      border-bottom: 0 !important;
+      padding-bottom: 0;
+    `}
```

Comment:
```suggestion
  ${p =>
    p.role &&
    ['parent', 'children'].includes(p.role) &&
    css`
    && {
      border-bottom: 0;
      padding-bottom: 0;
      }
    `}
```

maybe?

Reply:
Yep, that also did the trick. Applied here: https://github.com/getsentry/sentry/pull/27497